rules/linux/defense_evasion_c89_binary.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/03/15"
+maturity = "production"
+updated_date = "2022/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary c89/c99 abuse to break out from restricted environments by spawning an interactive system
+shell.This activity is not standard use with this binary for a user or system administrator. It indicates a potentially
+malicious actor attempting to improve the capabilities or stability of their access
```

Comment:
Maybe we can elaborate a less generic description, describe what the binary does, etc

Reply:
The description was more like a template for all the detections going out in terms of shell evasion for UNIX libraries. 
What could be captured in one more sentence is "The c89 utility is an interface to the standard C compilation system and it is uncommon for any user to spawn a shell from this utility" 

Can be paraphrased better and taken in code sanity before feature merge for all rules description.

Reply:
Yeah we may want to go back in and add another sentence to these rules defining the binary and it's normal usage.